lisa/env.py

```diff
@@ -210,6 +212,46 @@ def from_one_conf(cls, path):
             plat_info = None
         return cls(target_conf=target_conf, plat_info=plat_info)
 
+    @classmethod
+    def from_cli(cls):
+        """
+        Create a TestEnv from command line arguments
```

Comment:
The docstring should state that it should only be used to build simple throw away scripts, since even `using parse_know_args` instead of `parse_args` to let other parts of the script use the CLI will lead to confusing --help messages, unless a more complicated API is introduced (which defeats the orginial purpose of a helper for quick&dirty scripts).

Also, there can be argument clash if we add more here, and a helper from a library should not be in charge of defining the CLI of a tool built using it, so it really has to be restricted to quick scripts just like `fileinput` module intended use.